python/sidecar/settings.py

```diff
@@ -35,17 +39,46 @@ def make_settings_json_path(user_id: str) -> Path:
     return filepath
 
 
+def default_settings() -> FlatSettingsSchema:
```

Comment:
This is a way to define default settings without adding default values to the type.